### PR TITLE
[LDN-2023] activate proposal

### PR DESCRIPTION
**Pull request by andyburgin**

### CODE REVIEW

**Comment by glasnt** [comment]
The "Propose" in the navbar currently links directly to the form, not to the /propose page. (I can't immediately work out how to change this, but will update if I find it)

**Comment by andyburgin** [comment]
In data/events/2023-london.yml line 23 needs commenting out

See https://github.com/andyburgin/devopsdays-web/commit/f1d460cd60d303186883bdbd79e6ad1ca0e13788

I'd up date the PR but github only seems to let me compare ? maybe this needs merging/closing first ?

**Comment by mattstratton** [comment]
you don't have to make the propose link go to the page, you can directly link to the offiste link. there's a couple fields to make that work.

**Comment by mattstratton** [comment]
example: https://github.com/devopsdays/devopsdays-web/blob/55f1bcf9e2a2b6938b6a1fb83689e46b53e4dde2/data/events/2023-chicago.yml#L21

**Comment by mattstratton** [comment]
you can remove `propose.md` as long as everyone on that page is in your CFP that is linked; i see that it is not. the propose page includes "Due to the COVID-19 situation we are only looking for speakers who are normally a resident of the UK." which is not on the CFP form.

for the chicago event, for example, I keep the `propose.md` even though we have the navigation etc go to the CFP form, just in case someone types in `devopsdays.org/chicago/propose` but you don't need it if it is totally superflous

it does give you an easy to remember URL though

File: data/events/2023-london.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ startdate: 2023-09-21T00:00:00+01:00 # The start date of your event. Leave blank
...
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-#cfp_date_start: 2023-03-01T00:00:00+01:00 # start accepting talk proposals.
-#cfp_date_end: 2023-05-29T23:59:59+01:00 # close your call for proposals.
-#cfp_date_announce: 2023-07-10T23:59:59+01:00 # inform proposers of status
+cfp_date_start: 2023-01-31T00:00:00+01:00 # start accepting talk proposals.
+cfp_date_end: 2023-05-20T23:59:59+01:00 # close your call for proposals.
+cfp_date_announce: 2023-06-20T23:59:59+01:00 # inform proposers of status
 
-# cfp_link: "https://forms.gle/tbc"
+# cfp_link: "https://forms.gle/iBjCTD5N3abLAcNP8"
```
Comment (mattstratton) [inline data/events/2023-london.yml:23]: uncomment this out if you want the form to be the nav item; as i mentioned, please make sure that any requirements you have on `propose.md` are echoed there if you're going to skip that page!

**Comment by andyburgin** [comment]
closing, will recreate with correct settings

**Pull request closed without merging**